src/mainwindow.cpp

```diff
@@ -2739,6 +2739,9 @@ void MainWindow::readSettingsFromSettingsDialog(const bool isAppLaunch) {
             QStringLiteral("#eef067") : QStringLiteral("##141414")));
     }
 
+    const uint searchDebounceDelay = settings.value(QStringLiteral("searchDebounceDelay"), 0).toUInt();
```

Comment:
Don't forget that the delay in a QTimer is an int!

Reply:
static_casted the value to int in the call to `QTimer::setInterval`.